R/convertFlowCore.R

```diff
@@ -0,0 +1,177 @@
+library("stats")
+
+#' Convert a gate to flowCore
+#'
+#' Converts a CellEngine gate ("PolygonGate", "EllipseGate", or "RectangleGate") to
+#' its flowCore analogue.
+#'
+#' @param gate The CellEngine gate to be converted
+#' @export
+convertToFlowCore <- function (gate) {
```

Comment:
[`flowCore::transformList`](https://rdrr.io/bioc/flowCore/man/transformList-class.html) is their equivalent of our ScaleSet.

(Can be a later PR. Relates to #56.)

Reply:
Should this return the `transformList` equivalent to the `ScaleSet` for the experiment?

Reply:
Epic slow reply. Yes.

Reply:
It's nice to only have to remember two function names instead of many (the three in this PR, plus `transformListToScaleSet`, `compensation`↔`compensation`, `filterList`↔`population`, possibly more).

Another option could be to add S3 methods for these conversions, so `from.flowCore(flowCoreThing)` (or `as.cellengine` or something) would get overloads for each of the cases that's currently in the big `switch` block. CellEngine objects don't have classes assigned, so currently the other way wouldn't work, but we could change that.